source/simulator/parameters.cc

```diff
@@ -638,6 +649,14 @@ namespace aspect
                            "different value than described there: It can be chosen as stated there for "
                            "uniformly refined meshes, but it needs to be chosen larger if the mesh has "
                            "cells that are not squares or cubes.) Units: None.");
+        prm.declare_entry ("Discontinuous penalty", "10",
```

Comment:
I think you need to actually make use of this parameter in the assembly. IIRC, you're currently still hardcoding this value.